sw/device/lib/dif/dif_pinmux.c

```diff
@@ -0,0 +1,147 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/dif/dif_pinmux.h"
+
+#include <stdbool.h>
+#include <stddef.h>
+#include <stdint.h>
+
+#include "pinmux_regs.h"  // Generated.
+#include "sw/device/lib/base/mmio.h"
+
+// TODO - autogenerate?
```

Comment:
If these aren't generated then I'd file an issue and remove the TODO here, it will just rot over time. I also feel like `REGISTER_WIDTH` isn't a great description because it's more than one register in this bundle, right? Maybe `REGISTER_BLOCK_WIDTH`? Something to indicate that it's more than one register.

Reply:
Sure, I will fire an issue. I think both of the things you have mentioned in this comment are subject of a separate issue/PR, as this will include changes in reggen that will also affect other peripherals.

Reply:
Don't file an issue if this is covered by in-progress PRs (which I think it might be?)

Reply:
I think it should have been a part of #1814 but was missed. I have opened an issue and marked it as a good first issue, I don't think it is that important for the time being anyway.

Reply:
All TODOS have been removed in `.{c,h}`.

Reply:
Done.